src/Microsoft.AspNetCore.Razor.Tools/Program.cs

```diff
@@ -16,17 +17,31 @@ public static int Main(string[] args)
             var cancel = new CancellationTokenSource();
             Console.CancelKeyPress += (sender, e) => { cancel.Cancel(); };
 
+            var outputWriter = new StringWriter();
```

Comment:
Minor: the StringWriter is disposable, so should be wrapped with a using statement. However, this is in the main method, which means when it's unused, the process will exit and all the resources will be cleaned up anyway.